### PR TITLE
Add docs and example to onboard new benchmarks

### PR DESCRIPTION
**Pull request by pzread**
Add a README doc and an example model to show how to add and test new benchmarks.

### CODE REVIEW

File: common_benchmark_suite/README.md [hunk 1]
```diff
@@ -1,3 +1,320 @@
...
+4. To publish the benchmarks so they can be run in CI, generate and upload the
+    artifacts to GCS, then send out a PR for review.
+
+Based on the use cases, step 1 or 2 can be skipped if the definitions already
+exist. See the detailed steps below to see where to find and add definitions.
+
+Alternatively, you can check these commits to learn how an example benchmark is
+added:
+
+1. Add the base model implementation:
```
Comment (pzread) [inline common_benchmark_suite/README.md:20]: Reminder of update commits

File: common_benchmark_suite/README.md [hunk 2]
```diff
@@ -1,3 +1,337 @@
...
+| JAX                    | `[def_types.ModelArtifactType.STABLEHLO_MLIR, def_types.ModelArtifactType.XLA_HLO_DUMP]` |
+| PyTorch                | `[def_types.ModelArtifactType.LINALG_MLIR]` |
+| Tensorflow             | `[def_types.ModelArtifactType.STABLEHLO_MLIR, def_types.ModelArtifactType.XLA_HLO_DUMP, def_types.ModelArtifactType.TF_SAVEDMODEL_V2]` |
+
+You can see the example commit
+[e2b3566](https://github.com/openxla/openxla-benchmark/commit/e2b356659bd06039f454c5fac08ecabce4b80924)
+for how to populate these fields.
+
+After that, optionally add the new benchmarks to the benchmark list in one of
+`benchmark_all.sh` under [comparative_benchmark](/comparative_benchmark), so
```
Comment (mariecwhite) [inline common_benchmark_suite/README.md:332]: Might be useful to add a comment about how if the model is in JAX or Tensorflow, the model can also be added to the `benchmark_all.sh` script in xla-hlo for compiler-level benchmarks.

File: common_benchmark_suite/README.md [hunk 3]
```diff
@@ -1,3 +1,337 @@
...
+ def generate_default_inputs(self):
+   return "The quick brown fox jumps over a lazy dog"
+
+ def preprocess(self, input_text):
+   return self.tokenizer(input_text).repeat(self.batch_size)
+
+ def forward(self, batch_tokenized_input):
+   return self.model.forward(batch_tokenized_input)
+
+ def postprocess(self, output):
```
[the remark this answers is not in the document]
Comment (pzread) [reply common_benchmark_suite/README.md:93]: Removed.

File: common_benchmark_suite/README.md (hunk 2 above)
Comment (pzread) [reply common_benchmark_suite/README.md:331]: Done. I added comments about xla-hlo benchmarks below and the support of JAX and TensorFlow models at the beginning.

**Review by mariecwhite — APPROVED** [review]

**Pull request merged**